src/color-swatch/color-swatch.js

```diff
@@ -189,6 +180,35 @@ const Self = class ColorSwatch extends ColorElement {
 			},
 			dependencies: ["color"],
 		},
+		infoLabels: {
```

Comment:
Why do we need a separate infoLabels prop?

Reply:
Initially, I was thinking of optimizing the code so that we don't have to use `Object.entries` every time we need to get a channel and its label. It turned out that this was not a good idea: actually, we needed to do this in one place, so there was no need to add an extra prop. I removed it. Thank you for the question; it helped me see the flaw.